workflows/src/main/kotlin/hk/edu/polyu/af/bc/badge/flows/badgeclass/ReadBadgeClass.kt

```diff
@@ -0,0 +1,62 @@
+package hk.edu.polyu.af.bc.badge.flows.badgeclass
+
+import co.paralleluniverse.fibers.Suspendable
+import hk.edu.polyu.af.bc.badge.states.BadgeClass
+import net.corda.core.contracts.LinearPointer
+import net.corda.core.contracts.StateAndRef
+import net.corda.core.contracts.UniqueIdentifier
+import net.corda.core.flows.FlowException
+import net.corda.core.flows.FlowLogic
+import net.corda.core.flows.StartableByRPC
+import net.corda.core.flows.StartableByService
+
+
+/**
+ * Read all [BadgeClass].
+ *
+ */
+@StartableByRPC
+@StartableByService
+class ReadAllBadgeClass : FlowLogic<List<StateAndRef<BadgeClass>>>() {
```

Comment:
Do not use "Read". Use "Get"

Reply:
And it should be plural.

Reply:
Done

Reply:
​Do I need to change all the Read to Get or just change the GetAllBadgeClasses

Reply:
I will change all